hubblestack/files/hubblestack_nova/oval_scanner.py

```diff
@@ -32,27 +33,27 @@
     - cve.oval <-- assuming the yaml is oval.yaml in this example
 
 When run, the scanner will parse the source OVAL file into a readable
-dictionary, maps OVAL defintions directly to OVAL object and OVAL state
+dictionary, maps OVAL definitions directly to OVAL object and OVAL state
 references based on OVAL test reference data of the definition, and then makes
-a comparison to the local packages installed on the system to identify potential
-vulnerabilities.
+a comparison to the local packages installed on the system to identify
+potential vulnerabilities.
 
 This scanner currently only supports the Linux platform.
 """
 
+from __future__ import absolute_import
 
-
-import xml.etree.ElementTree as ET
```

Comment:
I'm uncomfortable introducing a new dependency. This looks like a smart choice for sure, but could you justify this briefly (just a reply in the PR is fine).

Reply:
The new dependency would be defusedxml.  This is based off of https://github.com/hubblestack/hubble/issues/942.  The idea is that we use defusedxml to parse the actual XML source .  Since defusedxml lacks an Element class, we still have to call on xml.etree.ElementTree.Element in order to verify Elements, but since the actual parsing of the XML is being done by defused, it should alleviate any vulnerability concerns.

Reply:
I admit I do like the idea. I just don't like bringing in another dependency.